src/System.Management.Automation/engine/InvocationInfo.cs

```diff
@@ -271,6 +271,18 @@ public string Line
             }
         }
 
+        /// <summary>
+        /// The full text of the invocation statement, may span multiple lines.
+        /// </summary>
+        /// <value>Statement that was entered to invoke this command.</value>
+        public string Statement
+        {
+            get
+            {
+                return ScriptPosition.Text;
+            }
+        }
```

Comment:
```suggestion
        public string Statement
        {
            get => ScriptPosition.Text;
        }
```

Reply:
Simplified getter probably not consistent with rest of the file 😄

Reply:
I deliberately kept style choices consistent with the existing code, yes ^^